runtime/src/main/java/org/corfudb/runtime/Backup.java

```diff
@@ -247,11 +272,11 @@ private void addToTarFile(File tableFile,TarArchiveOutputStream tarOutput) throw
     }
 
     /**
-     * Find tables which have the requires_backup_support tag set
+     * Get UUIDs of tables which have the requires_backup_support tag set
      *
      * @return  List<UUID>   - a list of UUIDs for all the tables which require backup support
      */
-    private List<UUID> getTaggedTables() {
+    private List<UUID> getTaggedTablesUUID() {
```

Comment:
previous name was better, since you use strongly typed language it's not recommended to use a type descriptor in the variable name

Reply:
Good idea! Done.